btk/survey.py

```diff
@@ -22,6 +22,18 @@
     ],
 )
 
+Survey.__doc__ = """
```

Comment:
Cool! I didn't know you could do this. Does it display the doc correctly in the auto doc?

Reply:
Yes it does ; you can actually do this with other python objects, it is equivalent to putting the docstring just after the function signature.